.nvidia-ci.yml

```diff
@@ -306,6 +342,25 @@ release:ngc-ubuntu22.04:
     - .dist-ubuntu22.04
     - .driver-versions
 
+release:ngc-ubuntu24.04:
+  extends:
+    - .release:ngc
+    - .dist-ubuntu24.04
+    - .driver-versions
```

Comment:
Note, we excluded this for the time being since we do not want to enable publishing of `ubuntu24.04` tags yet. Can we remove this and `release:ngc-precompiled-ubuntu24.04` until we have some QA coverage on these images? Let's just comment them out so we don't lose this work.

Reply:
corrected